src/ui/pages/SwapForm/SwapForm.tsx

```diff
@@ -620,28 +620,32 @@ export function SwapFormComponent() {
         ref={slippageDialogRef}
         height="360px"
         containerStyle={{ display: 'flex', flexDirection: 'column' }}
-        renderWhenOpen={() => (
-          <>
-            <DialogTitle
-              alignTitle="start"
-              closeKind="icon"
-              title={<UIText kind="headline/h3">Slippage</UIText>}
-            />
-            <Spacer height={24} />
-            <StoreWatcher
-              store={swapView.store.configuration}
-              render={(configuration) => (
-                <SlippageSettings
-                  configuration={configuration}
-                  onConfigurationChange={(value) => {
-                    swapView.store.configuration.setState(value);
-                    slippageDialogRef.current?.close();
-                  }}
-                />
-              )}
-            />
-          </>
-        )}
+        renderWhenOpen={() => {
+          invariant(chain, 'Chain must be defined');
```

Comment:
Can't this lead to an error for the user? Shouldn't we just wait until chain is not null?

Reply:
Ok, I see that the line that should call for this view shows only when chain is not null. So may be there is no problem

Reply:
Yes, we have similar invariant below, shouldn't be a problem